util/deploy-production.sh

```diff
@@ -116,11 +114,11 @@ ${UTIL_DIR}/docker-dev/run.sh -d
 # Login to gcloud if not already logged in.
 wptd_exec_it gcloud auth login
 # Deploy the services.
-wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=webapp/web
-wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=results-processor
-wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=api/query/cache/service
+wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=webapp/web ${QUIET:+QUIET=true}
+wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=results-processor ${QUIET:+QUIET=true}
+wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=api/query/cache/service ${QUIET:+QUIET=true}
```

Comment:
Don't you need to similarly update the [Makefile](https://github.com/web-platform-tests/wpt.fyi/blob/main/Makefile#L303) for this as well?

Reply:
`deploy.sh` actually already has a [quiet flag](https://github.com/web-platform-tests/wpt.fyi/blob/f6f4b7e138884f1b19136fa0f3c60e3f5b349287/util/deploy.sh#L13) implemented as well 🙂